results.html

```diff
@@ -219,4 +290,30 @@ <h5 class="white-text">Links</h5>
   <script src="./assets/js/results.js"></script>
 </body>
 
+<!-- footer -->
```

Comment:
I agree taking the `footer` out of `main` but it should still be inside the `body` tag according to [geeksforgeeks](https://www.geeksforgeeks.org/html5-footer-tag/)